ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -219,6 +238,10 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeStatsIndexEntriesPerSecond =
         verifiableProperties.getIntInRange("store.stats.index.entries.per.second", 240000, 1, Integer.MAX_VALUE);
     storeIndexPersistedEntryMinBytes = verifiableProperties.getInt("store.index.persisted.entry.min.bytes", 115);
+    storeReadOnlyEnableSizeThresholdPercentage =
+        verifiableProperties.getIntInRange(storeReadOnlyEnableSizeThresholdPercentageName, 95, 0, 100);
+    storeReadWriteEnableSizeThresholdPercentageDelta =
+        verifiableProperties.getIntInRange(storeReadWriteEnableSizeThresholdPercentageDeltaName, 5, 0, 100);
```

Comment:
shouldn't the range of this be [0, storeReadOnlyEnableSizeThresholdPercentage]?

Reply:
yeah good point.  changed.